src/DataFixtures/ORM/LoadCitizenActionData.php

```diff
@@ -59,6 +60,7 @@ public function load(ObjectManager $manager)
             'begin_at' => (new Chronos('+9 days'))->setTime(9, 00, 00, 000),
             'finish_at' => (new Chronos('+9 days'))->setTime(19, 00, 00, 000),
             'capacity' => 30,
+            'time_zone' => 'Europe/Paris',
```

Comment:
Isn't `'Europe/Paris'` the default timezone ?

Reply:
the timeZone he's not required for loadData